deployment/arm/deploy-global.sh

```diff
@@ -1,23 +1,36 @@
 #!/bin/bash
+set -eu
+
 parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$parent_path"
 
 businessUnit=$1
 appName=$2
 env=$3
+
+if [[ "$#" -ne 6 ]]; then
+    echo "Illegal number of arguments"
```

Comment:
I am being picky here - but if we tell the user they provided an illegal number of args, then we should provide some help/man to explain the expected args, right?